src/arguments.hpp

```diff
@@ -174,6 +174,17 @@ class Arguments
      */
     std::tuple<IpVer, std::string, uint8_t> asIpAddrMask();
 
+    /**
+     * @brief Get current argument as IP address or hostname.
+     *        Argument pointer will be moved to the next entry.
+     *
+     * @throw std::invalid_argument if there are no more arguments to handle
+     *                              or argument has invalid format
+     *
+     * @return IP address in unified format or hostname
+     */
+    std::string asRemoteAddress();
```

Comment:
The function name is confusing. May be, something like `asIpOrFqdn`?

Reply:
I agree and will rename it.

Reply:
done